src/log/syslog.rs

```diff
@@ -19,7 +19,32 @@ impl Log for Syslog {
         };
 
         let message = format!("{}", record.args());
-        syslog(priority, libc::LOG_AUTH, &message);
+        let message_len = message.bytes().len();
+
+        let mut x: usize = 960;
```

Comment:
I think `x` and `x_prev` could have more descriptive names like `end` and `start`. Maybe even a small comment explaining what they are.